LearnosityDemo/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:15545",
```

Comment:
I wonder if this means that on windows the port used is different... we should probably test this or change the port to 5000 to keep it consistent with non-windows platforms.

Reply:
Confirmed, Windows uses that port number to launch "IIS Express" on Windows. Tested, and updated to used port 5000 for consistency. https://github.com/Learnosity/learnosity-sdk-asp.net/commit/57c5fffa8793930658fe718805af14626b030b92